controls/datagrid/theming-and-styles/styling.md

```diff
@@ -21,6 +21,7 @@ The DataGrid control provides the following `Style` properties for customizing i
 * `HoverTextColor`&mdash;Defines the text color of the hovered cell.
 * `MouseHoverStyle`&mdash;Specifies the style applied to the cells and rows when the mouse is over them. The style is applicable for Desktop(MacCatalyst and WinUI).
 * `FrozenColumnsSplitterStyle`(`Telerik.Maui.Controls.Compatibility.DataGrid.DataGridFrozenColumnsSplitterStyle`) property allows you to style the splitter UI. The Splitter UI splits the frozen columns from the unfrozen columns.
+* `GridLinesVisibility`(`Telerik.Maui.Controls.Compatibility.DataGrid.RadDataGrid.GridLinesVisibility`) property defines which grid lines are currently visible (displayed). The property can be set to `Both`, `Horizontal`, `None` and `Vertical`.
```

Comment:
wrong namespace, add the correct one 

Add gridlines thickness and color properties

Reply:
The new commit resolves the comment.